FRAuth/FRAuth/Config/FROptions.swift

```diff
@@ -195,7 +195,29 @@ public class FROptions: NSObject, Codable {
     public func getEndSessionEndpoint() -> String {
         return self.endSessionEndpoint ?? "/oauth2/realms/\(self.realm)/connect/endSession"
     }
-    
+
+  /// Asynchronously discovers configuration options based on a provided discovery URL.
+  ///
+  /// - Parameter discoveryURL: The URL string from which to discover configuration options. This URL should point to a well-known configuration endpoint that returns the necessary configuration settings in a JSON format.
+  /// - Returns: An instance of `FROptions` populated with the configuration settings fetched from the discovery URL.
+  @available(iOS 13.0.0, *)
+  public func discover(discoveryURL: String) async throws -> FROptions {
```

Comment:
Should we make the FROptions `open` and such methods `open` as well to allow to override for special cases

Reply:
Made the class and discovery method open.